sdk/nodejs/appservice/zMixins_timer.ts

```diff
@@ -228,22 +228,60 @@ export class TimerSubscription extends mod.EventSubscription<TimerContext, Timer
 
         const { resourceGroupName, location } = mod.getResourceGroupNameAndLocation(args, undefined);
 
+        const func = new TimerFunction(name, args);
+
+        super("azure:appservice:TimerSubscription", name, func, {
+            ...args,
+            location,
+            resourceGroupName,
+        }, opts);
+
+        this.registerOutputs();
+    }
+}
+
+export type TimerFunctionArgs = util.Overwrite<mod.CallbackArgs<TimerContext, TimerInfo, void>, {
+    /**
+     * A schedule or a CRON expression for the timer schedule, e.g. '0 * * * * *'.
+     */
+    schedule: pulumi.Input<string | ScheduleArgs>;
+
+    /**
+     * If true, the function is invoked when the runtime starts.
+     */
+    runOnStartup?: pulumi.Input<boolean>;
+}>;
+
+/**
+ * Azure Function triggered on a CRON schedule.
+ */
+export class TimerFunction implements mod.FunctionArgs {
```

Comment:
why do we need this actual class?

Reply:
ah, i see now.  so the idea is that a user can make a MultiFunctionApp by passing in these classes.  And these classes are sometimes thin shims, but sometimes contain a lot of logic.  👍  on this.  


Note: we should probably doc at teh MultiFunctionApp level how you're supposed to do this.